plot_with_bigquery/plot_statvar_time_series.py

```diff
@@ -0,0 +1,116 @@
+# -*- coding: utf-8 -*-
+"""plot_statvar_time_series.ipynb
+
+Automatically generated by Colaboratory.
+
+Original file is located at
+    https://colab.research.google.com/drive/1rgAt2PkML8a4n4L1I-FzaMBtEiBo8qnj
+"""
+
+from google.colab import auth
+auth.authenticate_user()
+print('Authenticated')
+
+from google.cloud import bigquery
+project_id = "google.com:datcom-store-dev"
+client = bigquery.Client(project=project_id)
+
+obs_cols = [
+    'id', 'prov_id', 'is_public', 'observed_node_key', 'start_time_us',
+    'end_time_us', 'duration', 'measured_prop', 'sum_value', 'min_value',
+    'max_value', 'mean_value', 'median_value', 'measured_value', 'p10',
+    'p25', 'p75', 'p90', 'unit', 'sample_size', 'margin_of_error',
+    'std_error', 'mean_std_error', 'measurement_method', 'observation_date',
+    'observation_period', 'measurement_result', 'growth_rate', 'fake_date',
+    'compared_node_key', 'type', 'comparison_operator',
+    'std_deviation_value', 'location_key', 'measurement_denominator',
+    'scaling_factor', 'measurement_qualifier'
+]
+
+# Specify the DCID of your StatisticalVariable
+statvar_id = "US_PPI_Meats"
+
+# Define your axes
+x = "observation_date"
+y = "measured_value"
+sort_x = True
+group = ""
+
+triples_query = f'SELECT * FROM `google.com:datcom-store-dev.dc_v3_dev.Triple` WHERE subject_id = "{statvar_id}"'
```

Comment:
nit: worth define the table name as constant (also for the population and observation table)

Reply:
Yeah the code was really badly written. I need to find some time to fix it

Reply:
sg